homeassistant/components/backup/websocket.py

```diff
@@ -325,3 +322,23 @@ async def handle_config_update(
     changes.pop("type")
     await manager.config.update(**changes)
     connection.send_result(msg["id"])
+
+
+@websocket_api.require_admin
+@websocket_api.websocket_command({vol.Required("type"): "backup/subscribe_events"})
+@websocket_api.async_response
+async def handle_subscribe_events(
+    hass: HomeAssistant,
+    connection: websocket_api.ActiveConnection,
+    msg: dict[str, Any],
+) -> None:
+    """Subscribe to backup events."""
+
+    def on_event(progress: BackupEvent) -> None:
+        connection.send_message(websocket_api.event_message(msg["id"], progress))
+
+    manager = hass.data[DATA_MANAGER]
+    if manager.backup_event:
+        on_event(manager.backup_event)
```

Comment:
Is it expected to send the existing event before sending the result of the command?

Reply:
I also thought this was a problem, but @bramkragten has confirmed it's fine